packages/adminrouter/extra/src/docs/api/nginx.master.yaml

```diff
@@ -10,6 +10,9 @@ backends:
   dcos_net:
     server: '127.0.0.1:62080'
     name: DC/OS Net
+  dcos_ui_service:
```

Comment:
Can the name be changed to `dcos_ui_update_service` so it is clear that this is an update service that is different from the UI itself?